pipelines/sqre/infra/build_newinstall.groovy

```diff
@@ -37,6 +37,9 @@ notify.wrap {
   def baseDockerTag  = '7'
   def baseImage      = "${baseDockerRepo}:${baseDockerTag}"
   def splenvRef      = scipipe.canonical.lsstsw_config.splenv_ref
+  if (params.SPLENV_REF != "") {
+    spenvRef = params.SPLENV_REF
```

Comment:
"splenvref", no?

Reply:
Or `splenvRef` but yes that `l` is missing.